packages/edit-post/src/components/options-modal/index.js

```diff
@@ -105,6 +106,9 @@ export function OptionsModal( { isModalActive, isViewable, closeModal } ) {
 				</PageAttributesCheck>
 			</Section>
 			<MetaBoxesSection title={ __( 'Advanced panels' ) } />
+			<Section title={ __( 'Icon Labels' ) }>
+				<ShowIconLabels label="Show Icon Labels" />
```

Comment:
This string should be translatable.

Reply:
It should also be sentence case, right? 

Thank you for taking this on, by the way!

Reply:
The string has already been changed to "Show button text labels".

Reply:
Awesome. My mistake for giving a superficial review.